doc/ctc-meetings/2016-10-12.md

```diff
@@ -0,0 +1,158 @@
+  * Issue and PR review. A few PRs for tests.
+* Evan Lucas @evanlucas (CTC)
+  * Opened a few PRs
+  * Some issue/pr review
+  * Wrote Chrome extension to make generating review metadata easier
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+  * misc PRs / issues
+  * working with the ChakraCore team on ES Modules
+  * v6.8.0 Release
+* Michael Dawson @mhdawson (CTC)
+  * Misc review + land
+  * Back to working on adding nightly code coverage build
+  * ABI stable API PoC
+  * Keeping up to date on issues
+* Brian White @mscdex (CTC)
+  * Worked on improved string encoding/decoding performance
+  * Reviewed PRs, commented on issues
+* Ali Ijaz Sheikh @ofrobots (CTC)
+  * Travelling last week so not too much
+* Shigeki Ohtsu @shigeki (CTC)
+  * Review some PR and issues related to crypto.
+* Sakthipriyan Vairamani @thefourtheye (observer)
+  * catching up
+  * reviewing PRs and commenting
+* Trevor Norris (CTC)
+    * Finishing up implementing parentId for async hooks
+* Rich Trott @Trott (CTC)
+  * Issue and PR review
+  * governance discussions/issues
+  * Outreachy mentoring prep/work with applicants
+
+## Agenda
+
+Extracted from **ctc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* governance: expand use of CTC issue tracker [#8945](https://github.com/nodejs/node/pull/8945)
+* doc: add supported platforms list [#8922](https://github.com/nodejs/node/pull/8922)
+* net: multiple listen() events fail silently [#8419](https://github.com/nodejs/node/pull/8419)
+
+### nodejs/TSC
+
+* Consider folding TSC into CTC [#146](https://github.com/nodejs/TSC/issues/146)
+
+
+## Previous Meeting Review
+
+### nodejs/node
+
+* doc: add supported platforms list [#8922](https://github.com/nodejs/node/pull/8922)
+  * approved last week, if any objections comment now
+    on issue
+* Intl: Consider deprecating Intl.v8BreakIterator [#8865](https://github.com/nodejs/node/issues/8865)
+  * now closed so resolved
+* net: multiple listen() events fail silently [#8419](https://github.com/nodejs/node/pull/8419)
+  * back on agenda for this week
+
+### nodejs/CTC
+
+* Scheduling Meetings [#14](https://github.com/nodejs/CTC/issues/14)
+  * resolved, new meeting schedule set for next month.
+
+## Minutes
+
+### governance: expand use of CTC issue tracker [#8945](https://github.com/nodejs/node/pull/8945)
+
+  * don.t want to change decision making process without
+    putting through current process.
+  * ctc-review label would be useful.  For issues we need to
+    come to consensus but that we don.t need to bring to meeting.
+    Still good to open issues in CTC repo.
+  * Comment from @thefourtheye that section on consensus
+    seeking model could use some clarification. Rich -> likely
+    out of scope for this change.
+  * This only applies to issues that don.t need a vote, votes
+    mostly required when consensus cannot be achieved.
+  * Please add your comments or LTGM to the issue.
```

Comment:
There's three times `don.t` instead of `don't` in this paragraph. Is this intentional?